packages/manager/apps/veeam-backup/src/data/hook/useOrganizations.ts

```diff
@@ -14,3 +24,6 @@ export const getOrganizationIdFromBackupId = (backupId = '') =>
 
 export const getOrganizationDisplayName = (org: VCDOrganization) =>
   org?.currentState?.fullName || org?.currentState?.name;
+
+export const getBackupIdFromOrganizationId = (orgId = '') =>
+  `${orgId}-veeam-backup`;
```

Comment:
Is it really possible to not have any `orgId` here ?

Reply:
Currently it is used only in one place and it's not possible there